samples/fibonacci-action/README.md

```diff
@@ -4,8 +4,8 @@ This project is based on the Akka Serverless Maven archetype.
 
 ```shell
 mvn archetype:generate \
-  -DarchetypeGroupId=com.lightbend \
-  -DarchetypeArtifactId=maven-archetype-akkasls \
+  -DarchetypeGroupId=com.akkaserverless \
+  -DarchetypeArtifactId=akkaserverless-maven-archetype \
```

Comment:
This and the one below is not relevant. It's from the README in the samples. I'm just changing so I can rest at night. :-)